fabfile.py

```diff
@@ -148,9 +148,9 @@ def cfn_create():
         print 'Successfully built stack {0}.'.format(stack)
     else:
         print 'Failed to create stack: {0}'.format(stack)
-        #So delete the SSL cert that we uploaded
-        iam.delete_ssl_certificate(cfn_config.ssl(), stack_name)
-
+        #So delete the SSL cert that we uploaded (if it exists)
+        if 'ssl' in cfn_config.data:
+        	iam.delete_ssl_certificate(cfn_config.ssl(), stack_name)
```

Comment:
Two things. It also needs to create the iam object, should be like the create above. Also the indentation is not quite right.